tutorial/write-custom-widget.md

```diff
@@ -107,3 +107,186 @@ Take care not to keep any important state in your renderer - animation tickers
 are well suited to that location but user state would not be. A widget that is
 hidden may have it's renderer destroyed and if it is shown again the new renderer
 must be able to reflect the same widget state.
+
+### Example Widget
+
+The following covers in detail the creation of a simple Widget and its Renderer.
+
+#### Create a Widget
+
+As described above, we need to inherit from the `widget.BaseWidget` type.
+
+Go provides an simple mechanism for inheriting behaviour from other types. Adding an Anonymous type as a field (without a name) in the widget structure means that the widget inherits the behaviour of that Anonymous type.
+
+This also means that your widget now implements all of the public methods and inherits all of the public properties define by the  `widget.BaseWidget` type.
+
+We add an Anonymous `widget.BaseWidget` as follows:
+
+```go
+type MyWidget struct {
+	widget.BaseWidget        // Inherit from BaseWidget
+	text              string // The text to display in the widget
+}
+```
+
+To construct the widget create a New<widgetname> function. The example we are using is called MyWidget so by convention the following function name would be used to construct the widget:
+
+```go
+func NewMyWidget(text string) *MyWidget {
+	w := &MyWidget{
+		text: text,
+	}
+	w.ExtendBaseWidget(w)
+	return w
+}
+```
+
+Note : It is not necessary to override the Resize method (shown below) but if you need to change the Resize behaviour of your widget and you override the ```Resize(s fyne.Size)``` method then you **must** include the ```w.BaseWidget.Resize(s)```. in the code.
```

Comment:
```suggestion
Note: It is not necessary to override the Resize method (shown below) but if you need to change the Resize behaviour of your widget and you override the `Resize(s fyne.Size)` method then you **must** include the `w.BaseWidget.Resize(s)`. in the code.
```
This is a bit of a nitpick, but inline code blocks should only use one of those "things". This also applies to other parts in the code.

Reply:
That is a good catch, sorry I missed that one